bgpd/bgp_evpn_vty.c

```diff
@@ -477,6 +481,8 @@ static void display_vni(struct vty *vty, struct bgpevpn *vpn, json_object *json)
 				inet_ntoa(vpn->mcast_grp));
 		json_object_string_add(json, "advertiseGatewayMacip",
 				       vpn->advertise_gw_macip ? "Yes" : "No");
+		json_object_string_add(json, "advertiseSviMacip",
```

Comment:
Should this be enabled/disabled or Yes/No ?

Reply:
This seems to be already modified in the 2nd commit.

Reply:
The first commit adds a new field to existing JSON output, The second commit changes the behavior via setting active flag to per vni output when global knob is enabled. If local knob is enabled then display as "Enabled" value.
Since the behavior is change for existing field (defaultgw) and new filed (advertise-svi-ip) thus separate commit-id.